config/routes.rb

```diff
@@ -4,6 +4,8 @@
   namespace :api do
     post 'login', to: 'users#login'
     get 'feed', to: 'scores#user_feed'
-    resources :scores, only: %i[create destroy]
+    get 'users', to: 'users#index'
+    resources :scores, only: %i[create destroy show]
+    resources :users, only: %i[index show]
```

Comment:
Using `resources :users, only: %i[index show]` you already declare the route to `index`. No need for the manual route.